apps/diegesis-server/src/lib/makeServerHelpers/makeServerApp.js

```diff
@@ -23,7 +23,7 @@ function makeServerApp(config) {
             next();
         });
     }
-
+    app.use(cors({ origin: '*' }))
```

Comment:
We can't (or at least shouldn't) do this in production. Anything that turns of security features needs to be inside the "debug" option, just below, and I think that already does what you need. In other words, you just need debug: true in your config file.

Reply:
yes, I agree I have updated it intentionally to work with different ports and It should not be part of the PR will fix it.

Reply:
You can set ports in the config file. I've tried to make pretty much everything configurable. If I missed something, let's see if we can add it at the config level, rather than changing the codebase.